src/server/middleware/requestChangeMiddleware.js

```diff
@@ -19,7 +19,14 @@ module.exports = function requestChangeMiddleware(
 ) {
   const { platform } = req.query;
   if (platform) {
-    req.url = req.url.replace('index.bundle', `index.${platform}.bundle`);
+    /*
+      RN 0.52+ comes with experimental feature called Delta Bundles (from metro)
+      So to overcome any issues with this feature, one needs to disable it DevMenu
+    */
+    req.url = req.url.replace(
+      /index.(bundle|delta)/,
```

Comment:
Why do we rewrite the URL if we need to disable delta bundles? When it's disabled, will the request still come as `index.delta`? If it comes only when enabled, does serving the full bundle instead of delta bundle work?

Reply:
Left it here just in case someone would use debugger with delta enabled. I thought throwing an Error, telling to disable it, when requested `.delta` bundles instead. WDYT?

Reply:
Yes, we should throw an error asking to delta bundles. I guess with a response code 500 the device will show the error text in a Redbox